sale_order_line_price_history/wizards/sale_order_line_price_history.py

```diff
@@ -71,7 +72,12 @@ def _onchange_partner_id(self):
                 domain += [("order_partner_id", "child_of", self.partner_id.ids)]
 
         vals = []
-        order_lines = self.env["sale.order.line"].search(domain, limit=20)
+        sol_limit = (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("sale_order_line_price_history.order_line_limit", default="20")
```

Comment:
**praise:** ... and after all, you were already setting the default here. Cool! Then removing the ICP from data is even easier.